test.bat

```diff
@@ -29,6 +29,7 @@ goto finish
 echo.
 echo Usage: test.bat
 echo Usage: test.bat all - to run all tests
+echo Usage: test.bat full - to run all tests but notworking and ignored tests

```

Comment:
I am not sure of the description here, both say that they run all tests.  Should we mention here specifically the execution of integration tests?

Reply:
Yup, sounds good.

Reply:
Done